homeassistant/components/homee/config_flow.py

```diff
@@ -0,0 +1,123 @@
+"""Config flow for homee integration."""
+
+import logging
+from typing import Any
+
+from pyHomee import (
+    Homee,
+    HomeeAuthFailedException as HomeeAuthenticationFailedException,
+    HomeeConnectionFailedException,
+)
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.config_entries import ConfigFlowResult
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import AbortFlow
+
+from .const import CONF_ADD_HOMEE_DATA, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+BASE_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(
+            CONF_ADD_HOMEE_DATA,
+        ): bool,
```

Comment:
Okay so what was Homee data again?

Reply:
In Homee, a device consists of some common date for the device (device class, node-id, protocol,...) and then has a number of attributes representing the entities.
This data adds info about the attributes a device has, so one can see which capabilities it has, which may not yet be supported.
Thinking about it, we can drop this, since the logging code you want me to move to diagnostics platform gives more info on that.

Reply:
If you want you can even do diagnostics on device level, so you can just extract it per device (like if a usual setup has a lot of devices, might as well use that as addition to the entry wide diagnositcs because there it might be harder to find which device it is talking about)

Reply:
I will have to think about that when looking at the diagnostics platform.
I will also need to get information (or maybe that's the more important part) on nodes and attributes, that get not added to HA.
That is why I implemented the logging call in ```async_setup_entry```. But I will se  if the diagnostics can deliver what I need.